ddtrace/contrib/jinja2/patch.py

```diff
@@ -0,0 +1,80 @@
+import jinja2
+from wrapt import wrap_function_wrapper as _w
+
+from ddtrace import tracer
+
+from ...ext import http
+from ...utils.wrappers import unwrap as _u
+
+DEFAULT_TEMPLATE_NAME = "<template>"
```

Comment:
Please use single quotes instead of double quotes.

Our linter doesn't understand this style requirement yet 🙄